contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
@nomnombtc I think they're just saying that if you commit pre-generated files to the repo, users don't have to do it.

For a makefile target, i mean something like this: https://github.com/bitcoin/bitcoin/blob/master/src/Makefile.qt.include#L408

In this case, it would look something like (in src/Makefile.am):
```Makefile
update-man: $(bin_PROGRAMS)
       $(AM_V_at)echo "[COPYRIGHT]" > footer.h2m
       $(AM_V_at)echo "Copyright $(COPYRIGHT_YEAR) $(COPYRIGHT_HOLDERS_FINAL)" >> footer.h2m
       $(AM_V_at)for prog in $^; do help2man -N --version-string=$(PACKAGE_VERSION) --include=footer.h2m -o $(top_builddir)/doc/man/`basename $$prog`.1 $(builddir)/$$prog$(EXEEXT); done
```
(that's not really tested)

Since it's in a makefile, you have all of the vars at your disposal without trying to guess program names/paths.